Doc/using/cmdline.rst

```diff
@@ -1091,6 +1092,19 @@ conflict.
 
    .. versionadded:: 3.13
 
+.. envvar:: PYTHONFROZENMODULES
```

Comment:
as Hugo noted, we recently decided that new env vars should have `_` word separators, lets do that for this one.

Reply:
Updated to `PYTHON_FROZEN_MODULES`, please take a look.